zanata-client-commands/src/test/java/org/zanata/client/commands/init/SourceConfigPromptTest.java

```diff
@@ -109,4 +109,45 @@ public void allowUserToRetry() throws Exception {
                 Matchers.contains("*.*"));
         assertThat(opts.getExcludes(), Matchers.contains("a.pot"));
     }
+
+    @Test
+    public void canHandleFileProjectType() throws Exception {
+        when(fileClient.acceptedFileTypes()).thenReturn(ImmutableList.copyOf(DocumentType.values()));
+        // here we use absolute path because we create temp files in there
+        String expectedSrcDir = tempFolder.getRoot().getAbsolutePath() + "/resources";
+        ConsoleInteractor console =
+                MockConsoleInteractor.predefineAnswers(
+                        expectedSrcDir, "messages.md",
+                        "*Excluded.txt", "PLAIN_TEXT[md,txt]", "y");
```

Comment:
oh this test won't test that. It only test the input matches the option. I've tested it locally and it only works with ;